App/Course App/Course App/SharedUI/DesignSystem.swift

```diff
@@ -0,0 +1,100 @@
+//
+//  DesignSystem.swift
+//  Course App
+//
+//  Created by Christián on 21/05/2024.
+//
+
+import SwiftUI
+import UIKit
+
+enum FontSize: CGFloat {
```

Comment:
I would separate those types into own files. Specially view modifiers and pieces of design system as well.